### PR TITLE
Luke/autoenc cleanup

### PR DESCRIPTION
**Pull request by lukecwalters**
This includes some refactoring of the autoencoder experiments for minimal clean up.

### CODE REVIEW

File: experiments/autoencoders/autoencoders_simple.py [hunk 1]
```diff
@@ -13,6 +13,8 @@
 
 
 class DenseNet(nn.Module):
```
Comment (jonathanmei) [inline experiments/autoencoders/autoencoders_simple.py]: could we import this one from `.densenet` as well?

Comment (jonathanmei) [reply experiments/autoencoders/autoencoders_simple.py]: (and similarly the others)
```
from .configs import get_task_cfg
from .dataloaders import get_dataloaders
```

**Review by jonathanmei — COMMENTED** [review]
One small question, otherwise looks good!

File: experiments/autoencoders/autoencoders_simple.py (hunk 1 above)
Comment (lukecwalters) [reply experiments/autoencoders/autoencoders_simple.py]: yeah, I suppose we could.  I was originally thinking of making autoencoders_simple a standalone file.

Comment (jonathanmei) [reply experiments/autoencoders/autoencoders_simple.py]: gotcha. then we can go ahead and merge

**Review by jonathanmei — APPROVED** [review]

**Pull request merged**